core/embed/rust/src/ui/component/text/paragraphs.rs

```diff
@@ -23,6 +23,7 @@ pub struct Paragraphs<T> {
     area: Rect,
     list: Vec<Paragraph<T>, MAX_PARAGRAPHS>,
     placement: LinearPlacement,
+    theme: TextTheme,
```

Comment:
Using `&'static TextTheme` would make the struct smaller but not sure if it's not premature optimization.

Reply:
If we make the `TextStyle` reference in `TextLayout` &'static then `layout_ops` breaks because it mutates it. I don't see a simple fix aside from removing the reference from `TextLayout` entirely and pass it around separately.

Reply:
just make a copy of the font and color and mutate locally.
with the way `Op::skip_n_text_bytes` you should always get all the format mutating instructions at start, so the state of the locals will be the right thing before you get to any text

Reply:
I don't understand what you mean by mutate locally. `layout_ops` calls `layout_text` which reads the font and color from `TextLayout`. If we can't make up styles on the fly then we need to modify `layout_text` to accept font and color as a parameter?

Reply:
oooh right. OK let's leave that to the iterator PR